command/helper/config.go

```diff
@@ -33,6 +34,11 @@ type Config struct {
 	Join           string
 }
 
+// Tele
```

Comment:
Please add a better description of this struct

Reply:
Done, commit [link](https://github.com/0xPolygon/polygon-sdk/pull/216/commits/40e745fb9520b6e8aaae856a1f0a978803abe2ec)